Libs/MRML/Core/vtkMRMLMeasurement.h

```diff
@@ -153,26 +195,27 @@ class VTK_MRML_EXPORT vtkMRMLMeasurement : public vtkObject
   vtkMRMLMeasurement(const vtkMRMLMeasurement&);
   void operator=(const vtkMRMLMeasurement&);
 
+  /// Helper function to get unit node from the scene based on quantity name.
+  vtkMRMLUnitNode* GetUnitNode(const char* quantityName);
+
 protected:
   bool Enabled{true};
   std::string Name;
   double Value{0.0};
   bool ValueDefined{false};
-  std::string Units;
+  double DisplayCoefficient{1.0};  std::string Units;
   std::string Description;
   std::string PrintFormat; // for value (double), unit (string)
-  vtkCodedEntry* QuantityCode{nullptr};   // volume
-  vtkCodedEntry* DerivationCode{nullptr}; // min/max/mean
-  vtkCodedEntry* UnitsCode{nullptr};      // cubic millimeter
-  vtkCodedEntry* MethodCode{nullptr};     // Sum of segmented voxel volumes
+  vtkSmartPointer<vtkCodedEntry> QuantityCode;   // volume
+  vtkSmartPointer<vtkCodedEntry> DerivationCode; // min/max/mean
+  vtkSmartPointer<vtkCodedEntry> UnitsCode;      // cubic millimeter
+  vtkSmartPointer<vtkCodedEntry> MethodCode;     // Sum of segmented voxel volumes
   ComputationResult LastComputationResult{InsufficientInput};
 
-  /// Per-control point measurements
+  /// Per-control point measurements.
```

Comment:
Is this a style requirement?

Reply:
I don't think it is a strong requirement, but when there is no "." then the doxygen rendering looks like the text is trimmed or accidentally has not finished.

I'm not sure if somebody was just interrupted while writing the documentation and left it unfinished or that's the complete documentation:

![image](https://user-images.githubusercontent.com/307929/118191682-f3586680-b412-11eb-8a47-3580c854ebf0.png)

It is clear that this is all the documentation:

![image](https://user-images.githubusercontent.com/307929/118191690-f8b5b100-b412-11eb-9ba7-454eb6d88f34.png)